.github/workflows/blueprint.yml

```diff
@@ -0,0 +1,89 @@
+on:
+  push:
+    branches:
+      - master
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    name: Build project
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
```

Comment:
The actions used here have been deprecated. We should upgrade them to the latest version.
Alternatively, we could move this action to `build-strict.yml`, rename that file to `strict-blueprint-build.yml` (or similar) and have the blueprint job depend on the `build` job. That has the added bonus of removing duplicate code.